### ISSUE TITLE
User interfaces

### ISSUE THREAD

**Issue by Jwezenamo** [Jwezenamo/hello-world#1]
To make users enjoy interacting with your software and to like to spend a lot of quality time with it, then you've got to factor in the emotions the look, feel and use of your software brings to them whenever they encounter it.

For instance, the colour combination based on clour wave lengths should be in their right proportions. And also, the navigation must all be laid out and easy to figure out at first glance on your user interface. 

In summary, people shouldn't get irritated or nervous just for looking at your software user interface, and again, why should they have to think a lot just to figure out where to go next?
